src/js/angular/graphql/controllers/graphql-playground-view.controller.js

```diff
@@ -0,0 +1,18 @@
+const modules = [];
+
+angular
+    .module('graphdb.framework.graphql.controllers.graphql-playground-view', modules)
+    .controller('GraphqlPlaygroundViewCtrl', GraphqlPlaygroundViewCtrl);
+
+GraphqlPlaygroundViewCtrl.$inject = ['$scope', '$repositories'];
+
+function GraphqlPlaygroundViewCtrl($scope, $repositories) {
+    // not implemented
+    $scope.configuration = {
+        endpoint: '/rest/repositories/test/graphql/swapi'
```

Comment:
Shouldn't available endpoints be obtained via the backend API?

Reply:
sure, but this is the initial version